modules/ROOT/pages/advanced-typography.adoc

```diff
@@ -0,0 +1,60 @@
+= Advanced Typography plugin
+:navtitle: Advanced Typography
+:description: Automatically or interactively replace typewriter-centric characters with characters preferred by typesetters.
+:description_short: Auomatically or interactively typeset {productname} documents.
+:keywords: plugin, typography, typesetting, smart quotes, dashes, em-dash, en-dash, quotation marks
+:pluginname: Advanced Typography
+:plugincode: typography
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.3v.adoc[]
+
+The {pluginname} plugin, depending on its setting, automatically or interactively replaces common typewriter-style characters with their professional typesetter’s equivalents.
+
+For example, if English-language typography rules are set, the {pluginname} plugin can automatically replace the double-hyphen (`--`) with an em-dash (`—`). And it can automatically replace the teardrop apostrophe (`'`) with opening and closing quote marks (`‘` and `’`).
```

Comment:
> plugin can automatically replace the double-hyphen (--) with an em-dash (—)

This does not correspond to reality. Typography plugin can replace single hyphen `-` with an em-dash `—` in some languages, but we do not have any rules to replace double-hyphens.

Reply:
I’ve edited this accordingly but, as I noted elsewhere, this is a hole (I’d argue a gaping hole) in the rules.

The double-hyphen as em-dash substitute is proverbial.